src/HandleCors.php

```diff
@@ -56,7 +64,16 @@ public function handle($request, Closure $next)
             });
         }
 
-        $response = $next($request);
+        try {
+            $response = $next($request);
+        } catch (Throwable $e) {
+            if (! ($e instanceof Exception)) {
+                $e = new FatalThrowableError($e);
```

Comment:
This package doesn't depend on `symfony\debug`. Also, no such package exists for Symfony 5, so it will be impossible to use this code while being compatible with both Laravel 6 and 7.

Reply:
Ahh okay, will have to change that.